org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeSerialPort.java

```diff
@@ -170,7 +170,11 @@ public boolean open(int baudRate, FlowControl flowControl) {
                 logger.error("Serial Error: Too many listeners on Port {}.", portName);
                 return false;
             } catch (RuntimeException e) {
-                logger.debug("Device cannot be opened on Port {}. Caused by {}", portName, e.getCause());
+                if(e.getMessage().contains("No such file or directory in open") || e.getMessage().contains("No such device in open")) {
```

Comment:
Does this work with different languages?

Reply:
I believe this messages come from the native part and are hardcoded there.

Reply:
And is this text common for all JREs?  It just seems like a not very deterministic way to do this across all systems in all languages etc. I'm not really sure why this is needed to be honest?

So long as this doesn't continually log every few seconds in the event that the stick is removed, I don't see why we don't just log as error in the same way as the other errors logged here.

Also, can you make the text the same as the other errors (ie add "Serial Error: ").

Reply:
> So long as this doesn't continually log every few seconds in the event that the stick is removed, I don't see why we don't just log as error in the same way as the other errors logged here.

Unfortunately we have observed that (maybe depending on the serial library used underneath) that these RuntimeExceptions occur periodically when the stick is plugged out. Therefore we thought we "somehow" catch that case separately and log it on debug level, whereas other RuntimeExceptions should still be visible on the warn level.

So the question is: Are these messages localized somehow? I wonder where one could look that up...